### PR TITLE
Add structured logging and reduce log noisiness

### PR DESCRIPTION
**Pull request by NicolasPennie**
## Overview
Use structured JSON logging to allow log applications (e.g. CloudWatch) to query by fields (e.g. level). Adjust log levels to reduce overall log noisiness and storage requirements.

Also switched off-chain retrieval errors to separate log and metric. This scenario is expected and polluted metrics/logs, making it harder to detect more serious errors.

## Testing
Deployed e2e in Helius prod devnet and mainnet environments.

### CODE REVIEW

File: digital_asset_types/src/dapi/change_logs.rs [hunk 1]
```diff
@@ -1,5 +1,6 @@
 use sea_orm::sea_query::Expr;
 use sea_orm::{DatabaseConnection, DbBackend};
+use log::debug;
```
Comment (vovkman) [inline digital_asset_types/src/dapi/change_logs.rs:3]: why not just use tracing package everywhere?

File: nft_ingester/src/program_transformers/bubblegum/db.rs [hunk 2]
```diff
@@ -96,7 +97,7 @@ where
         // sequence number.  So in this case we set at flag to force checking the tree.
         let force_chk = rows.is_empty() && change_log_event.seq > 1;
 
-        println!("Adding to backfill_items table at level {}", i - 1);
+        info!("Adding to backfill_items table at level {}", i - 1);
```
Comment (vovkman) [inline nft_ingester/src/program_transformers/bubblegum/db.rs:100]: this needs to be info?

File: nft_ingester/src/program_transformers/mod.rs [hunk 3]
```diff
@@ -69,7 +69,7 @@ impl ProgramTransformer {
         &self,
         tx: &'a TransactionInfo<'a>,
     ) -> Result<(), IngesterError> {
-        println!("Handling Transaction: {:?}", tx.signature());
+        info!("Handling Transaction: {:?}", tx.signature());
```
Comment (vovkman) [inline nft_ingester/src/program_transformers/mod.rs:72]: this needs to be info?

File: nft_ingester/src/program_transformers/bubblegum/db.rs (hunk 2 above)
Comment (NicolasPennie) [reply nft_ingester/src/program_transformers/bubblegum/db.rs:100]: Could be debug. It's pretty rare and not very noisy.

File: nft_ingester/src/program_transformers/mod.rs (hunk 3 above)
Comment (NicolasPennie) [reply nft_ingester/src/program_transformers/mod.rs:72]: Quite useful for debugging yes. Volume isn't too high at the moment.

File: digital_asset_types/src/dapi/change_logs.rs (hunk 1 above)
Comment (NicolasPennie) [reply digital_asset_types/src/dapi/change_logs.rs:3]: Not necessary, can use the log facade.

**Review by vovkman — APPROVED** [review]

**Pull request merged**